packages/website/src/components/lib/jsonSchema.ts

```diff
@@ -50,7 +50,7 @@ export function getRuleJsonSchemaWithErrorLevel(
       };
     }
     // example: naming-convention rule
-    if (typeof ruleSchema.items === 'object' && ruleSchema.items) {
+    if (typeof ruleSchema.items === 'object') {
```

Comment:
```
typeof null === 'object'
```

Reply:
I think in the context of the types it's probably defined as `items?: T[]` - hence the `typeof` is enough to refine the types correctly?
This was a strictness change we added in v6.

Reply:
I don't see any `null` values for `items` in any ESLint or typescript-eslint rule. Is there a reason to support it?